### ISSUE TITLE
Make it clear that the value can be changed in MODX tables [ux/ui]

### ISSUE THREAD

**Issue by Ruslan-Aleev** [modxcms/revolution#14176]
In MODX tables, you can change the value by double-clicking, but this is not obvious to the user.

Need add a class for fields that can be changed by double-clicking and changing the cursor to "pointer" in this class.

![edit_cursor](https://user-images.githubusercontent.com/12523676/49375080-ba354b80-f71c-11e8-9fea-fec72f89a442.png)

Or besides changing the cursor add a tip using `<div ext:qtip="[[++settings_key]]<br>Double-click to edit">` for such fields (should also specify the [[++settings_key]] for setting fields, as well as in the resource fields).

![edit_3_cursor](https://user-images.githubusercontent.com/12523676/49511167-cef01b80-f8a3-11e8-9ee8-aa6776d5ea6a.png)

Together with https://github.com/modxcms/revolution/issues/14125, the user-friendly MODX will improve significantly.

**Comment by Ruslan-Aleev** [comment]
@Jako Can you help with advice or information?
If in the `/manager/assets/modext/widgets/core/modx.grid.settings.js` (script to build system settings) file can specify css at column "Value":
...
```
{
header: _('value')
,dataIndex: 'value'
,sortable: true
,editable: true
,renderer: this.renderDynField.createDelegate(this,[this],true)
,width: 260
,css: 'cursor: pointer; font-style: italic;'
}
```
...
But I did not understand how to add a class or tooltip.
Is it even possible in the current version of the script?

Here's how to look with css:

![settings_grid](https://user-images.githubusercontent.com/12523676/51902120-b1868c00-23d2-11e9-90cf-d079ac62d10d.gif)

**Comment by Jako** [comment]
The tooltip (or any other change of the column) could be created in the renderer property. The method in there has to return the display value. If I remember right, the method could return html tags, too.

**Comment by Mark-H** [comment]
Fixed in #14748
